tests/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+add_subdirectory(libdnf/conf)
```

Comment:
This line broke the build. There's no CMakeLists in that directory.

Reply:
Thanks :facepalm: I always forget to stage something.